src/components/app.js

```diff
@@ -29,37 +29,39 @@ class App extends React.Component {
         <main id='panel'>
           <Header />
           <div className='qpp-docs-title-container'>
-            <div className='content-container'>
+            <div className='ds-l-container'>
               <a className='qpp-docs-title-back' href='https://qpp.cms.gov/developers'>&lt; back to QPP Developer Tools</a>
               <h3 className='qpp-docs-title-text'>QPP Submissions API Documentation</h3>
             </div>
           </div>
 
-          <div className='content-container ds-l-row'>
-            <div id='dropdown-nav' className='ds-l-col--12 ds-u-padding-top--3'>
-              <form>
-                <DropdownNav />
-              </form>
-            </div>
+          <div className='ds-l-container'>
+            <div className='content-container ds-l-row'>
```

Comment:
Just flaggin `content-container` which looks like it's not needed anymore since the CSS rule-set got removed?

Reply:
Good catch, updated https://github.com/CMSgov/qpp-submissions-docs/pull/56/commits/2e61099bf4e5b297c9f207f32a1408542c2844fb